code.sh

```diff
@@ -60,5 +60,6 @@ for i in "${SDK[@]}"; do
   fi
 done
 
-exec env PATH="${PATH}:${XDG_DATA_HOME}/node_modules/bin" \
-  /app/extra/vscode/bin/code --extensions-dir=${XDG_DATA_HOME}/vscode/extensions "$@" ${WARNING_FILE}
+exec env ELECTRON_RUN_AS_NODE=1 PATH="${PATH}:${XDG_DATA_HOME}/node_modules/bin" \
```

Comment:
What about patching `/app/extra/vscode/bin/code` to use `zypak-wrapper`? It seems like a better solution than bypassing upstream's wrapper script.